lib/interfaces/src/IOExpander.cpp

```diff
@@ -0,0 +1,29 @@
+#include "IOExpander.h"
+#include "SharedFirmwareTypes.h"
+
+/* Returns intended watchdog state */
```

Comment:
What is "Returns intended watchdog state?" The IOExpander is not related to WD at all.